### ISSUE TITLE
One or more subtasks failed

### ISSUE THREAD

**Issue by mdehrzad** [PrismLauncher/PrismLauncher#2689]
### Operating System

macOS

### Version of Prism Launcher

8.4

### Version of Qt

i don't know

### Description of bug

https://mclo.gs/dcT99PF

### Steps to reproduce

https://mclo.gs/dcT99PF

### Suspected cause

https://mclo.gs/dcT99PF

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Trial97** [comment]
delete the folder found at:

fields:
  - name: Portable (Windows / Linux)
    value: '`./meta` In the PrismLauncher folder'
  - name: Windows
    value: '`%APPDATA%/PrismLauncher/meta`'
  - name: Scoop
    value: '`%HOMEPATH%\scoop\persist\prismlauncher\logs\PrismLauncher-0.log`'
  - name: macOS
    value: '`~/Library/Application Support/PrismLauncher/meta`'
  - name: Linux
    value: '`~/.local/share/PrismLauncher/meta`'
  - name: Flatpak
    value: '`~/.var/app/org.prismlauncher.PrismLauncher/data/PrismLauncher/meta`'
---

restart prism and try again 
If not working:

fields:
  - name: Portable (Windows / Linux)
    value: '`./logs/PrismLauncher-0.log` In the PrismLauncher folder'
  - name: Windows
    value: '`%APPDATA%/PrismLauncher/logs/PrismLauncher-0.log`'
  - name: Scoop
    value: '`%HOMEPATH%\scoop\persist\prismlauncher\logs\PrismLauncher-0.log`'
  - name: macOS
    value: '`~/Library/Application Support/PrismLauncher/logs/PrismLauncher-0.log`'
  - name: Linux
    value: '`~/.local/share/PrismLauncher/logs/PrismLauncher-0.log`'
  - name: Flatpak
    value: '`~/.var/app/org.prismlauncher.PrismLauncher/data/PrismLauncher/logs/PrismLauncher-0.log`'
---

Please send logs! The recommended site to upload your logs to is [mclo.gs](https://mclo.gs/).

**Comment by Mystris** [comment]
Hello, I seem to be having the same issue, however I am using Linux. I wasnt sure if its preferred to start a new issue or not, so apologies if I am putting this in the wrong place. 

I am using Linux Mint 21.3. Prism Launcher version 8.4 flatpak. The issue randomly occurred starting yesterday with no obvious trigger - I did not update the system or the launcher prior to the issue occurring, but to my knowledge they are both up to date. I was previously able to download mods and modpacks without issue. I am getting this "One or more subtasks failed" error when I try to download anything, such as modpacks, or individual mods from Curseforge or Modrinth. It also happens when I try to import modpacks from curseforge zip. The only instance I am able to import successfully is an instance that was exported from Prism itself as a "Prism Launcher (zip)" export.

I did try the instructions from the above comment to delete the meta folder, the issue was not resolved. Here is my log from mclo.gs:

[log](https://mclo.gs/HJmaYEs)

I also tried uninstalling and starting from a fresh reinstall. The issue persists.

Glancing at the end of the log shows it seems to be having some SSL Handshake errors, and my internet connection is stable and nothing else is having connectivity issues, I am not sure what else to check for that. Any help or suggestions would be appreciated.

**Comment by Trial97** [comment]
@Mystris you are most probably experiencing: https://github.com/PrismLauncher/PrismLauncher/issues/2704 
At the moment Modrinth is blocked by specific ISPs.

**Comment by Mystris** [comment]
@Trial97 thank you, the solution in that issue did seem to fix my issue as I was able to download and import modpacks while connected to a VPN. Interesting that it's only affecting Prism Launcher, but I am glad there is a workaround.

**Comment by Trial97** [comment]
If prism works with VPN that means something from your side is blocking prism requests (in the mentioned case the ISP is blocking the modrinth website)

**Comment by Trial97** [comment]
Closing this as no more info was provided.
I will reopen once the logs are provided from the original poster.

**Comment by Pokemastercz** [comment]
I am experiencing this on Windows 11
I do use the latest version of prismlauncher and i get precisely
this error when getting my mods.
Library mods work just fine but the main ones not so much.
The first thing didnt work and logs? OH if i had to run, it would be from that.

**Comment by Trial97** [comment]
@Pokemastercz  please open a new issue and make sure you post the mentioned logs.
